volumes.xconf

```diff
@@ -12,7 +12,12 @@
                     <!--<value>the</value>-->
                 </param>
             </analyzer>
-            <text qname="tei:div"/>
+            <text qname="tei:div">
+                <field name="within" expression="if (@type = ('section', 'document')) then 'documents' else ()"/>
+                <field name="content" expression="if (@type = ('section', 'document')) then . else ()"/>
```

Comment:
Since there's this `content` field containing the full text of the `<div>`, we could add `@index="no"` to the `<text qname="tei:div">`, and save eXist from a duplicate storage and processing of the div's contents, right?

Reply:
A corresponding change in the search module's KWIC handling would be to switch from `util:expand` to `ft:highlight-field-matches`.